compiler/lib/src/main/scala/analysis/UsedSymbols.scala

```diff
@@ -61,6 +67,8 @@ object UsedSymbols extends UseAnalyzer {
         case Symbol.EnumConstant(node) => defEnumConstantAnnotatedNode(a1, node)
         case Symbol.Module(node) => defModuleAnnotatedNode(a1, node)
         case Symbol.Port(node) => defPortAnnotatedNode(a1, node)
+        case Symbol.StateMachine(node) => defStateMachineAnnotatedNode(a1, node)
+        case Symbol.StateMachineInstance(node) => specStateMachineInstanceAnnotatedNode(a1, node)
```

Comment:
There should not be a symbol for a state machine instance. Only definitions get symbols.

Reply:
Fixed